lib/cmr/uvg_client.rb

```diff
@@ -9,5 +10,79 @@ def uvg_generate(params)
 
       post(url, params.to_json)
     end
+
+    def uvg_augment_definitions_stub(params)
+      url = if Rails.env.development? || Rails.env.test?
+              'http://localhost:4000/augment/definitions_stub'
+            else
+              '/augment/definitions_stub'
+            end
+
+      post(url, params.to_json)
+    end
+
+    def uvg_augment_keywords_stub(params)
+      url = if Rails.env.development? || Rails.env.test?
+              'http://localhost:4000/augment/keywords_stub'
```

Comment:
I see this code sprinkled all through mmt, if development || test do this else do that.   Why not just declare a host variable in env config to use instead?

Reply:
I'm not sure I follow what you're suggesting - can you give an example?

Reply:
I'm not suggesting change it (and there is probably reasons why it isn't this way), just wondering why this code can't be something like:
```
	      url = if Rails.env.development? || Rails.env.test?
	              'http://localhost:4000/generate_stub'
	            else
	      post(url, params.to_json)
	    end
```
```
post(Rails.application.config.generate_var_url, params.to_json)
```
To me seems like having specific service endpoints might be a good thing to have in config.

Reply:
I think we can discuss this further, but I'm not going to address it in this PR.

There are a lot of endpoints that we call, and I'm not sure how much of a benefit it would be to maintain a separate list of them, especially if they might be different between dev/test and the other environments. A big reason we have these is for the local cmr endpoints, which aren't always just a clean addition of some localhost port at the front.